docs/reference/config-params.md

```diff
@@ -656,9 +658,47 @@ services:
       kev.service.type: LoadBalancer
 ...
 ```
-
 #### None
 
+Simply, no service will be created.
+
+#### ClusterIP
+
+Choosing this type makes the Service only reachable internally from within the cluster by other services. There is no external access.
+
+In development, you can access this service on your localhost using [Port Forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/).
+
+It is ideal for an internal service or locally testing an app before exposing it externally.
+
+#### Nodeport
+
+This service type is the most basic way to get external traffic directly to your service.
+
+Its opens a specific port on all the Nodes (the VMs), and any traffic that is sent to this port is forwarded to the service.
```

Comment:
on all the Nodes (the VMs) -> on each of K8s cluster Nodes

forwarded to the service. -> forwarded to the ClusterIP service which is automatically created.